streams/src/test/java/org/apache/kafka/streams/processor/internals/tasks/DefaultTaskManagerTest.java

```diff
@@ -73,6 +80,8 @@ public void setUp() {
         when(task.id()).thenReturn(new TaskId(0, 0, "A"));
```

Comment:
nit: 
You could reuse `org.apache.kafka.test.StreamsTestUtils.TaskBuilder.statelessTask` here instead of creating your own stubs.

Reply:
Done